reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -151,22 +250,29 @@ private synchronized void refreshAperture() {
 
         long now = Clock.now();
         boolean underRateLimit = now - lastApertureRefresh > APERTURE_REFRESH_PERIOD;
-        int previous = targetAperture;
         if (avgPending < 1.0 && underRateLimit) {
```

Comment:
Is the idea here to refresh `pendings` EWMA even if we don't refresh the aperture?